src/Middleware/Rewrite/test/ApacheModRewrite/RewriteTokenizerTest.cs

```diff
@@ -91,5 +91,49 @@ public void Tokenize_AssertFormatExceptionWhenUnevenNumberOfQuotes()
             var ex = Assert.Throws<FormatException>(() => new Tokenizer().Tokenize("\""));
             Assert.Equal("Mismatched number of quotes: \"", ex.Message);
         }
+
+        [Fact]
+        public void Tokenize_CheckRegexDigitDoesNotHitUnescape()
```

Comment:
This could probably be a single parameterized test using `Theory` and `InlineData` (if you do a quick search there should be plenty of examples of how to do this.

Also, how about cases where there's a `\d` inside a more complicated Regex? Like `^/foo/(\\d)$`?

Reply:
The test is now a single parameterized test (thanks for pointing that out).
It did fail with more complicated regex but I have now fixed this in the latest commit.

Reply:
I suspected it might 😉